<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValBrush</class>
 <widget class="QWidget" name="ValBrush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Value Brush Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lIntensity">
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lSize">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lHardness">
      <property name="text">
       <string>Hardness</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="ctrlIntensity">
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="ctrlSize">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
      <property name="singleStep">
       <number>4</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="ctrlHardness">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.040000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value Brush Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
